pkg/sos/serviceFuncs.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package sos
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/gorilla/mux"
+)
+
+func RegistersNeccesaryPatterns(router *mux.Router) {
```

Comment:
for all these ones that have capital letters you need a comment in the form
// RegistersNecessaryFunctions does blah blah blah

Also Necessary not Neccesary :-)

Reply:
Fixed